src/status_im2/contexts/shell/scan/view.cljs

```diff
@@ -0,0 +1,274 @@
+    (if valid-connection-string?
+      (rf/dispatch [:syncing/input-connection-string-for-bootstrapping connection-string])
+      (rf/dispatch [:toasts/upsert
+                    {:icon           :i/info
+                     :icon-color     colors/danger-50
+                     :override-theme :light
+                     :text           (i18n/label :t/error-this-is-not-a-sync-qr-code)}]))))
+
+(defn- scan-qr-code-view
+  [qr-view-finder]
+  [:<>
+   [rn/view {:style style/scan-qr-code-container}]
+   (when (empty? @qr-view-finder)
+     [qr-scan-hole-area qr-view-finder])
+   (if (and @preflight-check-passed?
+            @camera-permission-granted?
+            (boolean (not-empty @qr-view-finder)))
+     [viewfinder @qr-view-finder]
+     [camera-and-local-network-access-permission-view])])
+
+(defn f-view
+  []
+  (let [insets         (safe-area/get-insets)
+        qr-view-finder (reagent/atom {})]
+    (fn []
+      (let [camera-ref                       (atom nil)
+            read-qr-once?                    (atom false)
+            ;; The below check is to prevent scanning of any QR code
+            ;; when the user is in syncing progress screen
+            user-in-syncing-progress-screen? (= (rf/sub [:view-id]) :syncing-progress)
+            on-read-code                     (fn [data]
+                                               (when (and (not @read-qr-once?)
+                                                          (not user-in-syncing-progress-screen?))
+                                                 (reset! read-qr-once? true)
+                                                 (let [timer-id (js/setTimeout (fn []
+                                                                                 (reset! read-qr-once?
+                                                                                   false))
+                                                                               3000)]
+                                                   (fn []
+                                                     (js/clearTimeout timer-id)))
+                                                 (check-qr-code-data data)))
+            show-camera?                     (and @camera-permission-granted?
+                                                  @preflight-check-passed?)
+            show-holes?                      (and show-camera?
+                                                  (boolean (not-empty @qr-view-finder)))]
+        (rn/use-effect
+         (fn []
+           (when-not @camera-permission-granted?
+             (permissions/permission-granted? :camera
+                                              #(reset! camera-permission-granted? %)
+                                              #(reset! camera-permission-granted? false)))))
+        [:<>
+         [blur/view
+          {:style         style/background-blur-overlay
+           :blur-amount   80
```

Comment:
`80` seems like a huge number.  I believe it's supposed to be `20`.

<hr />

We have a similar background blur design in the `Activity Center` screen. It uses blur amount `20`.

https://github.com/status-im/status-mobile/blob/e64ff76e0b8617475f759e282bdf669d409d81b8/src/status_im2/contexts/shell/activity_center/view.cljs#L218

https://github.com/status-im/status-mobile/blob/e64ff76e0b8617475f759e282bdf669d409d81b8/src/status_im2/contexts/shell/activity_center/style.cljs#L56-L59